### PR TITLE
Fix Nightly build are actually running on default API but showing as its running on google_apis

### LINKED ISSUE #3095: Nightly build are actually running on default API but showing as its running on google_apis
**Describe the bug**
We need to fixed the label since on nighlty we are running it on ```default API``` but label showing ```google_apis```

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Fixes #3095

### CODE REVIEW

**Comment by kelson42** [comment]
@gouri-panda

**Comment by kelson42** [comment]
@gouri-panda can we merge this?

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         api-level: [21, 22, 23, 24, 25, 27, 28, 30]
-        target: [google_apis]
+        target: [default]
```
Comment (gouri-panda) [inline .github/workflows/nightly.yml:14]: @MohitMaliDeveloper I still think `google_apis` are the best. If you want it then we can change it to default and see how nightly goes.
cc @kelson42

Comment (MohitMaliFtechiz) [reply .github/workflows/nightly.yml:14]: hi @gouri-panda actually by setting the target is not actualising running the ```google_apis``` while running the emulator , you can check https://github.com/kiwix/kiwix-android/actions/runs/3178395185/jobs/5179833538#step:5:26. So its was showing as on github as they are running ```google_apis``` but eventually its was ```default ``` one. ```google_apis``` are not stable they even fail for android 30 while you can see nightly works well with ```default```.

Comment (gouri-panda) [reply .github/workflows/nightly.yml:14]: Ok. Let's give them a try.

**Review by gouri-panda — APPROVED** [review]

**Pull request merged**